audio-worklet/src/bitcrusher-demo.js

```diff
@@ -22,12 +22,9 @@ class BitcrusherDemo {
    */
   constructor(containerId, bitDepth, reduction, gain) {
     this.context_ = new AudioContext();
-    this.masterGain_ = new GainNode(this.context_);
-    this.masterGain_.gain.value = gain || 0.5;
-
+    this.masterGain_ = new GainNode(this.context_, {gain: (gain || 0.5)});
```

Comment:
They are not necessary. I sometimes use parens to make code easier to read but I don't need to that.